nifi-nar-bundles/nifi-scripting-bundle/pom.xml

```diff
@@ -67,15 +67,27 @@
                 <version>1.10.0-SNAPSHOT</version>
                 <scope>provided</scope>
             </dependency>
+            <!--
+                Groovy is provided, by default, in a 'test' scope for use with unit tests
+                Explicitly specify Groovy dependencies as 'compile' to provide Groovy support in the
+                scripting processors
+            -->
             <dependency>
                 <groupId>org.codehaus.groovy</groupId>
-                <artifactId>groovy-all</artifactId>
-                <version>2.4.15</version>
-                <!--
-                    Groovy is provided, by default, in a 'test' scope for use with unit tests
-                    Explicitly provide this as 'compile' to provide Groovy support in the
-                    scripting processors
-                -->
+                <artifactId>groovy-json</artifactId>
+                <version>2.5.4</version>
+                <scope>compile</scope>
+            </dependency>
+            <dependency>
```

Comment:
See previous comment about other missing modules.

Reply:
Similar to the `nifi-groovyx-bundle` and `nifi-groovyx-processors` concerns, `nifi-scripting-bundle` and `nifi-scripting-processors` will be kept in sync regarding the inclusion of other modules that were included when `groovy-all` was used.